tests/cpu/Config_tests.cpp

```diff
@@ -5011,7 +5143,7 @@ R"([OpenColorIO Warning]: FixedFunction style is experimental and may be removed
 
         OCIO::ConstConfigRcPtr config;
         OCIO_CHECK_THROW_WHAT(config = OCIO::Config::CreateFromStream(is), OCIO::Exception,
-            "Only config version 2.4 (or higher) can have FixedFunctionTransform style 'ACES_OUTPUT_TRANSFORM_20'.");
+            "Only config version 2.4 (or higher) can have FixedFunctionTransform style 'ACES2_OutputTransform'.");
```

Comment:
@remia , I'm not sure why this was passing in the unit tests in your ACES PR.

Reply:
I was not using `FixedFunctionStyleToString(ffstyle)` to print the style in the original PR. But it's better the way you have done it!